scripts/download-python-packages.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python3
+
+import sys
+import subprocess
+from itertools import product
+
+if __name__ == "__main__":
+    requirements = sys.argv[1]
+    destination = sys.argv[2]
+
+    platforms = ["manylinux1_x86_64", "macosx_10_13_x86_64", "win32", "win_amd64"]
+    versions = ["35", "36", "37"]
+
+    tuples = product(platforms, versions)
+
+    base_cmd = ["python3", "-m", "pip", "download", "-d", destination, "-r", requirements]
+
+    subprocess.run(base_cmd + ["--platform", "any", "--no-deps"])
```

Comment:
Beware, the default behavior of `subprocess.run()` on failure is to not handle it. You can set [`check=True`](https://docs.python.org/3.5/library/subprocess.html#subprocess.run) if you want it to throw an exception on non-zero exit. Alternatively, you can handle it manually by inspecting the [`subprocessCompletedProcess`](https://docs.python.org/3.5/library/subprocess.html#subprocess.CompletedProcess) object that `run()` returns.